packages/annotation/src/utils/MathUtils.ts

```diff
@@ -4,6 +4,9 @@
 
 import { DEFAULT_FONT, DEFAULT_TEXT_MAX_WIDTH, ELineTypes, SEGMENT_NUMBER } from '@/constant/tool';
 import type { IPolygonData, IPolygonPoint } from '@/types/tool/polygon';
+import _ from 'lodash';
```

Comment:
```suggestion
import _ from 'lodash-es';
```
目前 lodash/fp 模块 需要安装 lodash，但非fp模块使用 lodash-es 对 vite 更友好